Scripts/ci-configure-build-test.sh

```diff
@@ -0,0 +1,95 @@
+#! /bin/bash
+
+set -eou pipefail
+
+kokkos_root="$1"
+kernels_root="$2"
+tutorials_src="$3"
+cpp_compiler="$4"
+build_type="$5"
+backend="$6"
+
+# These are exercises with CMakeLists.txt in Begin and Solution subdirectories
+# TODO: advanced_reductions seems broken
+# TODO: hpcbind does not use cmake
+# TODO: instances does not use cmake
+# TODO: parallel_scan seems broken
```

Comment:
Can you elaborate?

Reply:
It just means there were compilation errors when I tried to compile it in this framework. I didn't spend more than  30s looking at any of them. I figured fixing them up could be follow-on work.

Reply:
Yes, see #104.